includes/admin/class-wc-admin-post-types.php

```diff
@@ -450,28 +450,29 @@ public function render_product_columns( $column ) {
 	 * @param string $column
 	 */
 	public function render_shop_coupon_columns( $column ) {
-		global $post;
+		global $post, $the_coupon;
```

Comment:
Does this need to be a global? The only other reference I am seeing to `$the_coupon` is in wc-cart - it's not a global variable there/doesn't seem needed.